---a/net/minecraft/block/BlockCarrot.java
+++b/net/minecraft/block/BlockCarrot.java
@@ -3,18 +3,16 @@
 import net.minecraft.init.Items;
 import net.minecraft.item.Item;
 
-public class BlockCarrot extends BlockCrops
-{
+public class BlockCarrot extends BlockCrops {
+
     private static final String __OBFID = "CL_00000212";
 
-    protected Item getSeed()
-    {
+    protected Item getSeed() {
         return Items.carrot;
     }
 
-    protected Item getCrop()
-    {
+    protected Item getCrop() {
         return Items.carrot;
     }
-}
+}
